config/routes.rb

```diff
@@ -6,7 +6,10 @@
   namespace :admin do
     resources :blog_posts do
       get "/get_tags" => "blog_posts#get_tags", as: :get_tags, on: :collection
+      get 'page/:page', :action => :index, :on => :collection
```

Comment:
I added this from the Kaminari documentation, https://github.com/amatsuda/kaminari#creating-friendly-urls-and-caching.